apps/admin_panel/assets/README.md

```diff
@@ -1,2 +1,28 @@
-# admin-panel-v2
-A admin panel for e-wallet.
+# Admin Panel
+
+The OmiseGO Wallet admin panel for a provider to manage staff, users, roles and permissions,
+API access, tokens, transactions, etc.
+
+## Usage
+
+### Development
+
```

Comment:
The development mode allows you to develop the Admin Panel by running the Admin Panel locally, and letting it connect to an eWallet server elsewhere.

Reply:
My 2 cents:

Running the Admin Panel in development mode locally lets you modify the code and see changes in real-time. You may connect to any eWallet as a backend, running locally or remotely.